net/ntopng/pkg-descr

```diff
@@ -4,4 +4,21 @@ is based on libpcap and it has been written in a portable
 way in order to virtually run on every Unix platform, MacOSX
 and on Windows as well.
 
+
+Plugin Changelog
+================
+
+1.2
+
+* Changed data directory to avoid warning
+
+1.1
+
+* Add https support
+
+1.0
+
+* Initial release
```

Comment:
Just a thought I have a few times now: "initial release" is certainly true, but giving a list of initial version features is also nice. What do you think?

Reply:
Good idea, I'll try to keep it in mind for the next updates

Reply:
thanks 👍